packages/@lwc/integration-karma/test/mixed-shadow-mode/retargeting/index.spec.js

```diff
@@ -0,0 +1,31 @@
+import { createElement } from 'lwc';
+import SyntheticParent from 'x/syntheticParent';
+
+function test(name, selector) {
+    it(name, () => {
+        const elm = createElement('x-synthetic-parent', { is: SyntheticParent });
+        document.body.appendChild(elm);
```

Comment:
Should the element be removed after the test so that the document is clean?

Reply:
This is automated for us [here](https://github.com/salesforce/lwc/blob/7e5db0911c07d5b9f2dacf547f0b0fee25ba849f/packages/%40lwc/integration-karma/helpers/test-setup.js#L25-L53)